<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCribbageCounter</class>
 <widget class="QMainWindow" name="FormCribbageCounter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cribbage Counter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonCalculate">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCutCard">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cut Card</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>73</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutValues">
     <item>
      <widget class="QComboBox" name="comboBoxCard1">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCard2">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCard3">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCard4">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCutCard">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>J</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Q</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>92</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutSuits">
     <item>
      <widget class="QComboBox" name="comboBoxSuit1">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Hearts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamonds</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clubs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spades</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSuit2">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Hearts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamonds</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clubs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spades</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSuit3">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Hearts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamonds</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clubs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spades</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSuit4">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Hearts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamonds</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clubs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spades</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCutSuit">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Hearts</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diamonds</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clubs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spades</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidgetScores">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCribbage_Counter">
    <property name="title">
     <string>Cribbage Counter</string>
    </property>
   </widget>
   <addaction name="menuCribbage_Counter"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
